response.go

```diff
@@ -45,9 +47,33 @@ func checkResponse(resp *http.Response, headerCondition func(string) Issues) (he
 // To interpret `issues`, see the list of conventions in the
 // documentation for Issues.
 func PreloadableResponse(resp *http.Response) (header *string, issues Issues) {
-	return checkResponse(resp, PreloadableHeaderString)
+	return checkResponse(resp, PreloadableHeaderStringWrapper, preloadlist.Bulk1Year)
+}
+
+// EligibleResponse checks whether an resp has a single HSTS header that
+// passes the preload requirements.
+//
+// Iff a single HSTS header was received, `header` contains its value, else
+// `header` is `nil`.
+// To interpret `issues`, see the list of conventions in the
+// documentation for Issues.
+func EligibleResponse(resp *http.Response, policy preloadlist.PolicyType) (header *string, issues Issues) {
+	return checkResponse(resp, EligibleHeaderString, policy)
+}
+
+// EligibleResponse1Year calls EligibleResponse with a policy of 
+// "bulk-1-year". Function exists for testing purposes
+func EligibleResponse1Year(resp *http.Response) (header *string, issues Issues) {
+	return EligibleResponse(resp, preloadlist.Bulk1Year)
 }
 
+// EligibleResponse18Weeks calls EligibleResonse with a policy of
+// "bulk-18-weeks". Function exists for testing purposes
+func EligibleResponse18Weeks(resp *http.Response) (header *string, issues Issues) {
+	return EligibleResponse(resp, preloadlist.Bulk18Weeks)
+}
```

Comment:
Move these into the test file that uses them (and they shouldn't be exported, i.e. start with a capital letter).